java-manta-it/src/test/java/com/joyent/manta/client/MantaClientMetadataIT.java

```diff
@@ -26,24 +26,28 @@
  * Tests for verifying the behavior of metadata with {@link MantaClient}.
  *
  * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
+ * @author <a href="https://github.com/nairashwin952013">Ashwin A Nair</a>
  */
-@Test( groups = { "metadata" })
+@Test(groups = {"metadata"})
 public class MantaClientMetadataIT {
     private static final String TEST_DATA = "EPISODEII_IS_BEST_EPISODE";
 
     private MantaClient mantaClient;
 
     private String testPathPrefix;
 
-    @BeforeClass
-    @Parameters({"usingEncryption"})
-    public void beforeClass(@Optional Boolean usingEncryption) throws IOException {
+    @Parameters({"encryptionCipher"})
+    public void beforeClass(final @Optional String encryptionCipher) throws IOException {
```

Comment:
Refactoring `beforeClass()` -> `MantaClientMetadataIT()` class constructor which was originally raising the `NullPointerException`